aim/web/ui/src/services/models/metrics/metricsAppModel.ts

```diff
@@ -1270,6 +1270,17 @@ function setTooltipData(
   tooltipData = data;
 }
 
+function onRunCopy(): void {
```

Comment:
Maybe we should have another name for this method

Reply:
done